src/components/Radio/Radio.tsx

```diff
@@ -79,6 +94,11 @@ const Radio = React.forwardRef<HTMLInputElement, RadioProps>((props, ref?) => {
         {...attributes}
       />
       {labelText && showLabel && <Label htmlFor={radioID}>{labelText}</Label>}
+      {(helperText || (errored && errorText)) && showLabel && (
```

Comment:
I think you can replace `(helperText || (errored && errorText))` with `footnote` since `footnote`'s value comes from either `helperText` or `errorText`.

Reply:
Yes, that change worked well.